<launch>
    <param name="/use_sim_time" value="true"/>

    <!-- start rviz and stage -->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find socspioneer)/config/gmapping.rviz"/>
    <node name="stage" pkg="stage_ros" type="stageros" args="$(find socspioneer)/data/meeting.world"/>

    <!-- start slam on robot_0, the name coming from stageros's meeting.world -->
    <include file="$(find socspioneer)/launch/slam.launch">
        <arg name="robot_name" value="robot_0" />
    </include>
    <include file="$(find socspioneer)/launch/slam.launch">
        <arg name="robot_name" value="robot_1" />
    </include>

    <!-- start robotDetectionNode -->
    <node name="robot_detector" pkg="socspioneer" type="robotDetectNode.py"/>
</launch>